compiler/passes/insertWideReferences.cpp

```diff
@@ -1581,7 +1586,8 @@ static void propagateField(Symbol* sym) {
               case PRIM_GET_SVEC_MEMBER_VALUE:
                 if (fIgnoreLocalClasses || !sym->hasFlag(FLAG_LOCAL_FIELD)) {
                   DEBUG_PRINTF("\t"); debug(lhs, "widened gmv\n");
-                  matchWide(use, lhs);
+                  // this was matchWide(use, lhs); -- Ben, FIXME
+                  matchWide(sym, lhs);
```

Comment:
It works correctly. Ben likes to have a slightly different style of widen-ing for debug printing etc as I understand it. This is a bug fix that we can't merge without, though.